middleware/cache/config.go

```diff
@@ -54,6 +54,11 @@ type Config struct {
 
 	// Deprecated, use KeyGenerator instead
 	Key func(*fiber.Ctx) string
+
+	// allows you to store additional headers generated by next middlewares & handler
+	//
+	// Default: false
+	E2EHeaders bool
```

Comment:
if i were to look at the flag based on the description of what it does, i would choose a different name, because it has nothing to do with end2end test and it only confuses consumers

sure it solves your problem what you had with the end2end tests, but it does the job of saving all headers and thats how it should be named

Reply:
> if i were to look at the flag based on the description of what it does, i would choose a different name, because it has nothing to do with end2end test and it only confuses consumers

Indeed the name is confusing 😕 
I wish RFC writers used a different naming convention to define HTTP headers categories. Because here I'm referring to end-to-end headers, not end-to-end tests. _(more: [end-to-end headers vs hop-by-hop headers](https://datatracker.ietf.org/doc/html/rfc2616#section-13.5.1))_.

How do you feel if I rename to `StoreResponseHeaders`?

Reply:
its better 
> How do you feel if I rename to StoreResponseHeaders?